### ISSUE TITLE
oneTimeCharge lässt sich nicht aktivieren

### ISSUE THREAD

**Issue by oliver976** [thovid/ioBroker.viessmannapi#57]
Ich habe versucht, oneTimeCharge per Java Script mit folgendem Befehl zu aktivieren:

sendTo('viessmannapi.0', 'action', { feature: 'heating.dhw.oneTimeCharge.active', action: 'activate', payload: {} });

Leider ohne Erfolg. "Ich möchte Warmwasser" wird nicht aktiviert. Hat jemand eine Idee, was ich falsch mache?

**Comment by xdaamg** [comment]
`            sendTo('viessmannapi.0', 'action', {
                feature: 'heating.dhw.oneTimeCharge',
                action: 'activate',
                payload: {}
            });`

Probier es mal so.

**Comment by tminimax** [comment]
Hi, 
i have the same issue. 'I tried the following command:
![grafik](https://user-images.githubusercontent.com/72692458/95677547-8227ab00-0bc6-11eb-9722-c071e42abe42.png)
a warning will apper shortly after the call in the log: "FeatureAction[activate];not executagle
the log looks like this:
![grafik](https://user-images.githubusercontent.com/72692458/95677568-a2576a00-0bc6-11eb-8b0c-6d2e9f49735a.png)

Other commands (e.g. adjusting the run-mode "heating.circuits.0.operating.modes.active") will work perfectly!
![grafik](https://user-images.githubusercontent.com/72692458/95677627-04b06a80-0bc7-11eb-9f5a-755d3df0e64c.png)

thank you for any help.

**Comment by oliver976** [comment]
> `sendTo('viessmannapi.0', 'action', { feature: 'heating.dhw.oneTimeCharge', action: 'activate', payload: {} });`
> 
> Probier es mal so.

Hat funktioniert! Danke!

**Comment by tminimax** [comment]
dürfte nun funktionieren (oder hat dann kurz funktioniert), aber nun gibt es ein neues Problem - siehe neues ISSUE...

**Comment by oliver976** [comment]
Ja, stimmt leider. Es wird zwar angezeigt, dass eine Einmalladung Warmwasser produziert wir. Doch es passiert nichts.

**Comment by thovid** [comment]
Habe das Command gerade getestet. Script mit 

`sendTo('viessmannapi.0', 'action', { feature: 'heating.dhw.oneTimeCharge', action: 'activate', payload: {} });`

gestartet, dann in der Vicare App geschaut: der "Ich möchte Warmwasser" - Modus ist aktiviert.

**Comment by oliver976** [comment]
Ja, das stimmt. Bei mir auch. 

Hast du dann aber mal nachgesehen, ob die Heizung dann auch tatsächlich mit der Aufbereitung des Warmwassers beginnt? Bei mir leider nicht. Die Heizung macht also nicht das, was in der App angegeben ist.

**Comment by tminimax** [comment]
Hallo,
also bei mir schaltet sowohl die VCareApp als auch der Heizungsregler selbst um und bereitet das Warmwasser auf!
Ev. den Adapter bzw. die Instanz mal neu starten - hatte ich auch mal...

**Comment by oliver976** [comment]
Habe den Adapter neu gestartet und es funktioniert.

Danke!!!
